src/Sylius/Bundle/ResourceBundle/spec/Routing/ResourceLoaderSpec.php

```diff
@@ -491,7 +664,10 @@ function it_generates_routing_for_a_section(
                 'permission' => false,
             ],
         ];
-        $routeFactory->createRoute('/products/', $indexDefaults, [], [], '', [], ['GET'])->willReturn($indexRoute);
+        $routeFactory
```

Comment:
Let's not do this kind of coding standard changes in a feature PR, it makes the real changes less clear.